iOS/issue-tracker/issue-tracker/View/IssueList/IssueDetailTableViewCell.swift

```diff
@@ -83,4 +82,19 @@ class IssueDetailTableViewCell: UITableViewCell {
             maker.leading.trailing.top.bottom.equalToSuperview().inset(20)
         }
     }
+
+    func configure(model: Comment) {
+        setupProfile(url: model.author.imageURL)
+        author.text = model.author.name
+        comment.text = model.content
+    }
+
+    private func setupProfile(url: String) {
+        DispatchQueue.global().sync {
+            guard let imageData = try? Data(contentsOf: URL(string: url)!) else { return }
```

Comment:
말안해도 아시죠? 옵셔널 강제 언래핑 이제 하시면 안돼요. URL은 더더욱이요 
해도 되는 단 한가지는 로컬에 이미지를 내장한 경우 UIImage 객체는 강제 언래핑해도 됩니다. 
로컬에 이미지 저장한 것이 확실하고 저장안되어 있다면 테스트할때 바로 잡을 수 있으니까요.


말씀드린 이미지 내용은 Swift Error Handling의 내용입니다. 

```
Disabling Error Propagation
Sometimes you know a throwing function or method won’t, in fact, throw an error at runtime. On those occasions, you can write try! before the expression to disable error propagation and wrap the call in a runtime assertion that no error will be thrown. If an error actually is thrown, you’ll get a runtime error.

For example, the following code uses a loadImage(atPath:) function, which loads the image resource at a given path or throws an error if the image can’t be loaded. In this case, because the image is shipped with the application, no error will be thrown at runtime, so it’s appropriate to disable error propagation.

let photo = try! loadImage(atPath: "./Resources/John Appleseed.jpg")
```